src/types/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface Todo {
+  id: number;
+  userId: number;
+  title: string;
+  completed: boolean;
+}
+
+export type SendedTodo = Omit<Todo, 'id'>;
+
+export type LoadTodos = () => void;
+
+export enum FILTERS {
+  all = 'All',
```

Comment:
Pls [uppercase for enum properties](https://www.typescriptlang.org/docs/handbook/enums.html)